packages/neutrino/bin/start.js

```diff
@@ -9,7 +9,7 @@ module.exports = (middleware, args, cli) => {
     cli,
     middleware,
     args,
-    NODE_ENV: 'development',
+    NODE_ENV: process.env.NODE_ENV || 'development',
```

Comment:
These are defaults, which get overwritten here:
https://github.com/armenzg/neutrino-dev/blob/7b9e6a9d63a17bab8e3aa7b35cfb7c5ea96ff324/packages/neutrino/bin/base.js#L21

I believe you want NODE_ENV to propagate via `args.options` and so the fix would need to be in `packages/neutrino/src/api.js` instead.